.github/workflows/tests.yml

```diff
@@ -0,0 +1,153 @@
+name: Simulator tests
+
+on:
+  push: {}
+  release:
+    types: [ created ]
```

Comment:
Why are we running this when creating releases too?

If the intention is to make publishing better, I'd suggest creating a dedicated workflow for it which only runs on release "publish", and uses https://github.com/RealOrangeOne/date-group/blob/master/.github/workflows/release.yml#L44

This could then be done instead of the archiving stuff below which creates double zips, or I guess as well as if we want to be sure it's working correctly.

Reply:
Is it worth also disabling run-match on releases?